Ginger/GingerCoreNET/PlugInsLib/PluginsManager.cs

```diff
@@ -69,16 +69,23 @@ public class DriverInfo
 
         public void AddPluginPackage(string folder)
         {
-            // Verify folder exist
-            if (!System.IO.Directory.Exists(folder))
+            try
             {
-                throw new Exception("Plugin folder not found: " + folder);
-            }
+                // Verify folder exist
+                if (!System.IO.Directory.Exists(folder))
+                {
+                    throw new Exception("Plugin folder not found: " + folder);
+                }
 
-            PluginPackage pluginPackage = new PluginPackage(folder);
-            pluginPackage.PluginPackageOperations = new PluginPackageOperations(pluginPackage);
-            pluginPackage.PluginPackageOperations.LoadPluginPackage(folder);
-            mSolutionRepository.AddRepositoryItem(pluginPackage);
+                PluginPackage pluginPackage = new PluginPackage(folder);
+                pluginPackage.PluginPackageOperations = new PluginPackageOperations(pluginPackage);
+                pluginPackage.PluginPackageOperations.LoadPluginPackage(folder);
+                mSolutionRepository.AddRepositoryItem(pluginPackage);
+            }
+            catch (Exception ex)
```

Comment:
add Error message on UI in Status Column

Reply:
updated